app/mailers/wishlist_mailer.rb

```diff
@@ -1,12 +1,13 @@
 class WishlistMailer < ApplicationMailer
-  def wish_list_creation_email(user)
-    @user = user
+  #he purpose of the wishlist creation email is to give confirmation to the therapist that the wishlist was successfully created
+  def wish_list_creation_email(donor)
+    @donor = donor
     @url  = login_url
-    mail(to: @user.email, subject: 'Congratulations for creating a wishlist!')
+    mail(to: @donor.email, subject: 'Congratulations for creating a wishlist!')
   end
-  def item_purchased_email(user)
-    @user = user
+  def item_purchased_email(donor)
+    @donor = donor
     @url  = login_url
-    mail(to: @user.email, subject: 'You have purchased an item you reserved')
+    mail(to: @donor.email, subject: 'You have purchased an item you reserved')
```

Comment:
Should I get rid of the email body?

->
h1 You have successfully purchased an item you reserved for a snowman.
p We have been notified that your purchase was successful. You will shortly be receiving details on when the item will be delivered to your secret snowman.

p Thanks for your support and have a great day!